packages/core/lib/console.js

```diff
@@ -272,7 +273,7 @@ class Console extends EventEmitter {
       // Theoretically stderr can contain multiple errors.
       // So let's just print it instead of throwing through
       // the error handling mechanism. Bad call?
-      console.log(spawnResult.stderr.toString());
+      debug(spawnResult.stderr.toString());
```

Comment:
[spawnSync](https://nodejs.org/dist/latest-v16.x/docs/api/child_process.html#child_processspawnsynccommand-args-options) returns `status`, `signal` and `error` components in its return object. Do you think it's worth putting the debug behind a guard that checks iff there was an error? It might even be worth it to include those values when logging

Reply:
We would get more info this way, right? If that's the case I don't think we should.

Reply:
There could be more information, but it would only be presented if DEBUG environment is configured